radicle-cli/src/commands/merge.rs

```diff
@@ -71,51 +69,50 @@ pub enum State {
     Open,
     Merged,
 }
+
 #[derive(Debug)]
 pub struct Options {
-    pub id: Rev,
+    pub revision_id: Rev,
+    pub force: bool,
     pub interactive: bool,
-    pub revision: Option<RevisionIx>,
 }
 
 impl Args for Options {
     fn from_args(args: Vec<OsString>) -> anyhow::Result<(Self, Vec<OsString>)> {
         use lexopt::prelude::*;
 
         let mut parser = lexopt::Parser::from_args(args);
-        let mut id: Option<Rev> = None;
-        let mut revision: Option<RevisionIx> = None;
+        let mut force = true;
```

Comment:
`force` flags usually default to `false`

Reply:
Yup, not good.  Had to rebase to update this with recent changes, and [corrected the above patch](https://github.com/radicle-dev/heartwood/pull/461/commits/1dd4d7f777845deb18c5551b987a68154d5c804f)

Reply:
:+1: I've rebased + squashed due to merge conflicts against master.  The [valule in question has been fixed](https://github.com/radicle-dev/heartwood/pull/461/commits/8603bfcab4f1b105727338b7374d55ffe8ed6c9d)